File: Beginner/Snack_1038/main.py
snacks = {
    "1": 4,
    "2": 4.5,
    "3": 5,
    "4": 2,
    "5": 1.50
}

splitted_values = input().split(" ")
code, amount = splitted_values[0], float(splitted_values[1])

total = snacks[code] * amount
print(f'Total: R$ {total:.2f}')

File: Beginner/Snack_1038/Main.java
import java.util.HashMap;
import java.util.Hashtable;
import java.util.Map;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        Map<String, Double> snacks = new HashMap<String, Double>();
        snacks.put("1", 4.0);
        snacks.put("2", 4.5);
        snacks.put("3", 5.0);
        snacks.put("4", 2.0);
        snacks.put("5", 1.5);

        
        String values = sc.nextLine();
        String[] splittedValues = values.split(" ");
        String code = splittedValues[0];
        double amount = Double.parseDouble(splittedValues[1]);
        Double total = snacks.get(code) * amount;
        
        System.out.printf("Total: R$ %.2f\n", total);
        sc.close();
    }
}
